project-maintainers.csv

```diff
@@ -1053,15 +1053,13 @@ Sandbox,Devfile,Andreas Resios,AWS,resios, https://github.com/devfile/api/blob/m
 ,,Ida Olsen,AWS,miaida,
 ,,Stevan Le Meur,Red Hat,slemeur,
 ,,Tomas Kral,Red Hat,kadel,
-Incubating,Knative,Carlos Santana,AWS,csantanapr, https://github.com/knative/community/blob/main/OWNERS_ALIASES
-,,Lance Ball,Red Hat,lance,
+Incubating,Knative,Ali Ok,Red Hat,aliok, https://github.com/knative/community/blob/main/OWNERS_ALIASES
 ,,Adolfo García Veytia,Chainguard,puerco,

```

Comment:
we should add @krsna-m

Reply:
and @davidhadas

Reply:
Hm, but what would be the selection then? If we also add WG leads, shouldn't also Client, Docs, UX and so on be listed?